Server/JokesOnYou.Web.Api/Authentication/ApplicationUser.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+
+namespace JokesOnYou.Web.Api.Authentication
+{
+    public class ApplicationUser : IdentityUser
```

Comment:
We have a User already, are you using the latest dev branch?

Reply:
I pulled down and merged on Saturday. I'll pull down fresh now.

Reply:
@chrisK00 pulled fresh and merged into my branch. No changes. Quite a few of these comments reflect are also for code I had no hand in changing. Not sure if I should nuke this branch

Reply:
> @chrisK00 pulled fresh and merged into my branch. No changes. Quite a few of these comments are for code I had no hand in changing. Not sure if I should nuke this branch

Weird, to me it does look old when i compared with dev branch. Do you think its better to copy the code youve written over to a new branch?

Reply:
@chrisK00 yeah, I'll create a new branch now with my changes.  Thanks :)